libraries/matchers.rb

```diff
@@ -26,8 +26,8 @@ def delete_graphite_storage(name)
     ChefSpec::Matchers::ResourceMatcher.new(:graphite_storage, :delete, name)
   end
 
-  def create_graphite_carbon_conf_accumulator(name)
-    ChefSpec::Matchers::ResourceMatcher.new(:graphite_carbon_conf_accumulator, :create, name)
+  def create_socrata_graphite_fork_carbon_conf_accumulator(name)
```

Comment:
You can delete this entire file if you want. ChefSpec automatically generates matchers now instead of making you define your own.